examples/e2e-tests/kubernetes/kubernetes-config/addons-disabled.json

```diff
@@ -0,0 +1,56 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "kubernetesConfig": {
+        "addons": [
+          {
+            "name": "tiller",
+            "enabled": false
+          },
+          {
+            "name": "aci-connector",
+            "enabled": false
+          },
+          {
+            "name": "kubernetes-dashboard",
+            "enabled": false
+          },
+          {
+            "name": "rescheduler",
+            "enabled": false
+          }
+        ]
+      }
+    },
+    "masterProfile": {
+      "count": 5,

```

Comment:
I still think we should do single masters for these tests, thoughts?

Reply:
Yes that's a miss, I'll fix it! same for addons-enabled